src/githubHelper.ts

```diff
@@ -1175,14 +1183,6 @@ export class GithubHelper {
       );
     }
 
-    // Store usernames found in the text
-    const matches: Array<string> = str.match(usernameRegex);
-    if (matches && matches.length > 0) {
-      for (const username of matches) {
-        this.users.add(username.substring(1));
-      }
-    }
-
```

Comment:
This way the `users.txt` won't contain the users that matched the mapping, which makes more sense (we want the users before the mapping happens, exactly to help us to create the list for the mapping :slightly_frowning_face: ). 

Thank you for fixing the logic, @LuiggiTenorioK !